generator.py

```diff
@@ -57,7 +57,7 @@
         getSettingsFunc = "function getSettings() {\n\tlet jsonObj = {}\n"
 
         for item in config["toggles"]:
-            toggleTable += "<td> <input type=\"checkbox\" id=\"{0}Toggle\" title=\"{1}\"> <label for=\"{0}Toggle\" title=\"{1}\">{2}</label> </td>\n".format(item["id"], item["desc"], item["name"])
+            toggleTable += "<td onclick=\"{0}Toggle.click()\"> <input type=\"checkbox\" id=\"{0}Toggle\" title=\"{1}\"> <label for=\"{0}Toggle\" title=\"{1}\">{2}</label> </td>\n".format(item["id"], item["desc"], item["name"])
```

Comment:
I don't think this onclick is ever used, the "getSettings()" function just polls the checkboxes directly, theres no tracking of the state in the background

Reply:
the onclick just checks the checkbox so you don't have to be sniper accurate to toggle an option